packages/makefile/action.yaml

```
@@ -126,7 +126,7 @@ runs:
     - name: Setup yq
       uses: chrisdickinson/setup-yq@latest
       with:
-        yq-version: v4.11.2
+        yq-version: v4.29.1
```

```suggestion
        yq-version: v4.30.6
```

Ini latest versionnya @dimzrio